install-dev/upgrade/sql/1.7.7.0.sql

```diff
@@ -12,3 +12,7 @@ ALTER TABLE `PREFIX_supply_order_detail` ADD `mpn` VARCHAR(40) NULL AFTER `upc`;
 ALTER TABLE `PREFIX_stock` ADD `mpn` VARCHAR(40) NULL AFTER `upc`;
 ALTER TABLE `PREFIX_product_attribute` ADD `mpn` VARCHAR(40) NULL AFTER `upc`;
 ALTER TABLE `PREFIX_product` ADD `mpn` VARCHAR(40) NULL AFTER `upc`;
+
+/* Set optin field value to 0 in employee table */
+ALTER TABLE `PREFIX_employee` MODIFY COLUMN `optin` tinyint(1) unsigned NOT NULL DEFAULT 0;
```

Comment:
```suggestion
ALTER TABLE `PREFIX_employee` MODIFY COLUMN `optin` tinyint(1) unsigned DEFAULT NULL;
```
At least the developers can save old values. (Which newer having a null value)
Or it's mayb useless

Reply:
@PierreRambaud  I think we don't want those values to be saved.

Reply:
But if we apply the patch of the PR, no value can't be saved.